kubernetes/README.md

```diff
@@ -0,0 +1,24 @@
+# Deploying with Kubernetes
+
+## TL;DR
+
+```console
+$ git clone git@github.com:jmbannon/ytdl-sub.git
+$ cd ytdl-sub/kubernetes
+$ $EDITOR kustomization.yaml subs.yaml presets.yaml
```

Comment:
Can we rename these to subscriptions.yaml and config.yaml, to keep consistent with the documentation